README.md

```diff
@@ -1,5 +1,9 @@
 # testctxlint
 
+<p align="center">
+  <img src="assets/logo.svg" alt="testctxlint logo" width="300">
```

Comment:
Can we just use the circle logo here and put the text "testctxlint" as a title next to it? It doesn't need to be part of the svg.